tests/worker_coordinator/runner_test.py

```diff
@@ -2232,29 +2237,30 @@ async def test_create_pipeline(self, opensearch):
 
         await r(opensearch, params)
 
-        opensearch.ingest.put_pipeline.assert_called_once_with(id="rename", body=params["body"], master_timeout=None, timeout=None)
+        opensearch.ingest.create_ingest_pipeline.assert_called_once_with(id="rename", body=params["body"], master_timeout=None,
+                                                               timeout=None)
 
     @mock.patch("opensearchpy.OpenSearch")
     @run_async
     async def test_param_body_mandatory(self, opensearch):
-        opensearch.ingest.put_pipeline.return_value = as_future()
+        opensearch.ingest.create_ingest_pipeline.return_value = as_future()
 
         r = runner.CreateIngestPipeline()
 
         params = {
             "id": "rename"
         }
         with self.assertRaisesRegex(exceptions.DataError,
-                                    "Parameter source for operation 'put-pipeline' did not provide the mandatory parameter 'body'. "
+                                    "Parameter source for operation 'create-ingest-pipeline' did not provide the mandatory parameter 'body'. "
                                     "Add it to your parameter source and try again."):
             await r(opensearch, params)
 
-        self.assertEqual(0, opensearch.ingest.put_pipeline.call_count)
+        self.assertEqual(0, opensearch.ingest.create_ingest_pipeline.call_count)
 
     @mock.patch("opensearchpy.OpenSearch")
     @run_async
     async def test_param_id_mandatory(self, opensearch):
-        opensearch.ingest.put_pipeline.return_value = as_future()
+        opensearch.ingest.create_ingest_pipeline.return_value = as_future()
```

Comment:
We won't want to change these because this is an opensearchpy function call. We're only changing occurrences of "put-pipeline" in OSB. Let me know if you have more questions about this and we can discuss this on a call or during office hours.